dot/services.go

```diff
@@ -280,7 +280,7 @@ func (nodeBuilder) createNetworkService(cfg *Config, stateSrvc *state.Service,
 		LogLvl:            cfg.Log.NetworkLvl,
 		BlockState:        stateSrvc.Block,
 		BasePath:          cfg.Global.BasePath,
-		Roles:             cfg.Core.Roles,
+		Roles:             common.Roles(cfg.Core.Roles),
```

Comment:
nit do you think we could have the `cfg.Core.Roles` directly as `common.Roles` type?

Reply:
with latest commit, roles are getting used everywhere instead of byte. I have propagated these changes til toml config. I would not like to change toml config, since that's what deals with marshalling and marshalling config. I don't want to disturb any behaviour there.

Reply:
Having a byte alias probably won't change config parsing. Don't we have unit tests to verify this?